docs/documenting/standards.md

```diff
@@ -12,19 +12,19 @@ To ensure that the documentation for Home Assistant is consistent and easy to fo
 - Be objective and not gender favoring, polarizing, race related or religion inconsiderate.
 - The case of brand names, services, protocols, integrations and platforms must match its respective counterpart. e.g., "Z-Wave" **not** "Zwave", "Z-wave", "Z Wave" or "ZWave". Also, "Input Select" **not** "input select" or "Input select".
 - Do not use ALL CAPITALS for emphasis - use _italics_ instead.
+- All examples containing Jinja2 templates should be wrapped **outside** of the code markdown with the `{% raw %}` tag.
```

Comment:
Do we need to state this only applies to home-assistant.io docs

Reply:
Maybe, I just moved it from another section. I do agree this page needs more love, but tried to keep it out of scope as much as possible.

Reply:
👍